Description: systemd: Use high priority
 Use -N to raise the priority of ntpd.
Forwarded: https://gitlab.com/NTPsec/ntpsec/merge_requests/575
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2017-11-08
--- a/etc/ntpd.service
+++ b/etc/ntpd.service
@@ -10,7 +10,7 @@
 [Service]
 Type=forking
 PrivateTmp=true
-ExecStart=/usr/bin/ntpd -g -u ntp:ntp
+ExecStart=/usr/bin/ntpd -g -N -u ntp:ntp
 Restart=always
 
 [Install]
--- a/etc/rc/ntpd
+++ b/etc/rc/ntpd
@@ -13,7 +13,7 @@
 PIDFILE=/var/run/ntpd.pid
 USER=ntp
 GROUP=ntp
-NTPD_OPTS="-g -u $USER:$GROUP -p $PIDFILE"
+NTPD_OPTS="-g -N -u $USER:$GROUP -p $PIDFILE"
 
 ntpd_start() {
     if [ -r $PIDFILE ]; then
